packages/p2p/src/utils/adverts.js

```diff
@@ -23,3 +24,61 @@ export const generateErrorDialogBody = (error_code, error_message) => {
     }
     return error_message;
 };
+
+export const getLastOnlineLabel = (is_online, last_online_time) => {
+    if (!is_online) {
+        if (last_online_time) {
+            const start_time = epochToMoment(last_online_time).unix();
+            const end_time = toMoment().unix();
+            const diff = getDiffDuration(start_time, end_time);
+
+            if (diff.years()) return localize('Seen more than 6 months ago');
+            if (diff.months()) {
+                if (diff.months() > 6) {
+                    return localize('Seen more than 6 months ago');
+                }
+                if (diff.months === 1) {
+                    return localize('Seen {{ duration }} month ago', {
+                        duration: diff.months(),
+                    });
+                }
+                return localize('Seen {{ duration }} months ago', {
```

Comment:
@farhan-nurzi-deriv Couldn't use `Moment.js`'s `Relative Time`? 🤔

Reply:
but can it be translated later if i pass it to `localize` function? since its dynamic 🤔

Reply:
@farhan-nurzi-deriv It's been a long time I haven't use `Moment` actually 😅 But if I remember correctly `Moment` itself contains the translation for all languages so we don't need to translate anything 🤔 

You can ignore this comment, It's just a suggestion 🙇🏻

Reply:
Haa i see. I saw that i can use locale from `Moment`. But, in this use case i need to add `Seen ...` prefix. So, i think still need to pass it to `localize()`